src/puppetlabs/puppetdb/command.clj

```diff
@@ -69,7 +69,7 @@
             [puppetlabs.puppetdb.utils :as utils]
             [slingshot.slingshot :refer [try+ throw+]]
             [puppetlabs.puppetdb.mq-listener :as mql]
-            [puppetlabs.puppetdb.command.constants :refer [command-names]]
+            [puppetlabs.puppetdb.command.constants :as cmd-consts]
```

Comment:
I'm more attached to the `q/foo` changes, since I found it hard to remember what's in the DLO vs. what's in the queue namespace, and the `q/` alias is very short. I'd be fine with dropping the new alias in here, though, if that's the consensus.